src/sentry/runner/commands/run.py

```diff
@@ -461,17 +461,36 @@ def post_process_forwarder(**options):
     type=click.Choice(["earliest", "latest"]),
     help="Force subscriptions to start from a particular offset",
 )
+@kafka_options(
+    "occurrence-consumer",
```

Comment:
Shouldn't this be query subscription consumer?

Reply:
That's what I get for copy pasting